packages/@lwc/style-compiler/src/serialize.ts

```diff
@@ -6,7 +6,7 @@
  */
 import postcss, { Result, Declaration } from 'postcss';
 import postcssValueParser from 'postcss-value-parser';
-
+import matchAll from 'string.prototype.matchall';
```

Comment:
Sadly we support Node v10 in `node-tool`, where `String.prototype.matchall` is [not available](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll#browser_compatibility).

Reply:
We aren't supporting Node v10 as it is already OEL. What makes you think it is the case?

Reply:
@pmdartus I got the impression from [this line](https://git.soma.salesforce.com/lwc/node-tool/blob/9ad011daf876f50da0ca48753115cd69c29dc0f8/pom.xml#L21) that we are supporting Node v10.